proposals/nnn-atf-sdl-watchdog-service.md

```diff
@@ -102,7 +102,8 @@ N/A
 
 ## Impact on existing code
 
-Impacts ATF code, some scripts
+Impacts atf component named SDL, also impacts Fasade of ATF. 
+Impacts all scripts that related to multiple ignition cycles 
```

Comment:
Can you please update this to "Impacts all scripts related to multiple ignition cycles."?

Reply:
@theresalech done with  3f9ec02